brain/circuit.cpp

```diff
@@ -396,36 +459,64 @@ neuron::Morphologies Circuit::loadMorphologies( const GIDSet& gids,
                                                 const Coordinates coords ) const
 {
     const URIs& uris = getMorphologyURIs( gids );
```

Comment:
Can you double check what does this function do when a gid doesn't exist in both mvd2 and mvd3?